src/views/Online/OnlineGameView.jsx

```diff
@@ -82,6 +89,28 @@ export function OnlineGameView({ ...props }) {
     }
   };
 
+  const scoreBoard = (gameData) => {
+    return (
+      <div className="flex flex-col gap-4">
+        <h1 className="text-primary text-4xl font-bold">🥳 GAME OVER 🥳</h1>
```

Comment:
Könntest du in ne eigene Komponente stecken

Reply:
👍